monitor.py

```diff
@@ -849,3 +850,37 @@ def delete(self, collection):
         """
         collection.delete()
 ReaperMonitor.REGISTRY.append(CollectionReaper)
+
+
+class CirculationEventScrubber(ReaperMonitor):
+    """Scrub location information from old CirculationEvents.
+
+    Unlike the other ReaperMonitors, this class doesn't delete rows
+    from the database -- it only clears out the 'location' column for
+    such rows.
+    """
+    MODEL_CLASS = CirculationEvent
+    TIMESTAMP_FIELD = 'start'
+    MAX_AGE = datetime.timedelta(days=365)
+
+    def run_once(self, *args, **kwargs):
+        rows_scrubbed = 0
+        qu = self.query()
+        self.log.info("Scrubbing %d row(s)", qu.count())
+        for i in qu:
+            self.scrub(i)
+            rows_scrubbed += 1
+        return TimestampData(achievements="Items scrubbed: %d" % rows_scrubbed)
```

Comment:
Most reapers don't implement run_once(), and this one only does so that we don't issue misleading messages about "Items deleted" or have a misleadingly-named delete() method.